training-maquetado/scss/navbar.scss

```diff
@@ -0,0 +1,26 @@
+.navbar {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  max-width: 100%;
+  height: 90px;
```

Comment:
There was an already defined variable for the height of the navbar at the beginning of the training